mydemoproject/demoapp1/urls.py

```diff
@@ -19,15 +19,25 @@
 from .views.inventory import InventoryListView
 from .views.book import (BookCreateView,
                          BookListView,BookUpdateView,
-                         BookDeleteView)
-from .views.borrowed import BorrowReturnView, BorrowedBookListView
-from .views.user import UserUpdateView, UserListView, UserDeleteView
+                         BookDeleteView, UserBookListView,
+                         )
+from .views.borrowed import (BorrowReturnView, BorrowedBookListView
```

Comment:
Use absolute imports.

Reply:
Maintain order of imports:

1. Third-party packages
2. 1 line space
3. Internal imports

All imports should be in alphabetical order.